src/JSHandle.js

```diff
@@ -15,6 +15,7 @@
  */
 
 const {helper, assert, debugError} = require('./helper');
+const path = require('path');
```

Comment:
Nit: since this is only used in `uploadFile`, let's move the `require` there. That way, scripts that don't use `uploadFile` don't need to pay the cost.

Reply:
Done.